salt/journal/config/etc-nginx-traits.d-redirect-existing-paths.conf

```diff
@@ -1444,4 +1444,6 @@ map $uri $new_uri {
     '/privacy-policy' '/privacy';
     '/terms-and-conditions-of-use' '/terms';
     '/highlights' 'https://crm.elifesciences.org/crm/highlights';
+    '/inside-elife/d3748fd9' '/inside-elife/690ef4c1';
+    '/inside-elife/d3748fd9/webinar-report-private-funding-in-science' '/inside-elife/690ef4c1';
```

Comment:
Should this be a wildcard redirect?

Reply:
The logs (`SELECT url, COUNT(*) FROM prod.journal WHERE url LIKE '/inside-elife/d3748fd9/%' GROUP BY url`) say:

Row | url | f0_ |  
-- | -- | -- | --
1 | /inside-elife/d3748fd9/webinar-report-private-funding-in-science | 62 |  
2 | /inside-elife/d3748fd9/elife-latest-looking-for-a-new-editor-in-chief | 2 |  
3 | /inside-elife/d3748fd9/something | 4

3 is me doing a test, 2 probably also is us doing some tests because it's clearly wrong. So no need for wildcards.